libraries/Servo/src/Servo.cpp

```diff
@@ -95,11 +90,8 @@ void Servo::detach()
 {
   if (_attached) {
     _servoMap &= ~(1 << _pin);
-#ifdef WAVEFORM_LOCKED_PHASE
+    // TODO - timeHigh == 0 is illegal in _PWM code branch. And now what?
```

Comment:
Sorry, forgot to remind you do drop this comment.  IT's done w/your last commit.

Reply:
oh sure, my bad